cmd/kubeadm/app/cmd/phases/init/uploadcerts.go

```diff
@@ -39,6 +39,7 @@ func NewUploadCertsPhase() workflow.Phase {
 		InheritFlags: []string{
 			options.CfgPath,
 			options.UploadCerts,
+			options.CertificateKey,
```

Comment:
we might also want to use the value of this flag instead of `copycerts.CreateCertificateKey()` in `runUploadCerts()`?

Reply:
`runUploadCerts()` only use `copycerts.CreateCertificateKey()` if `initDatra.CertificateKey was empty

Reply:
that's true!
i've missed that.